x/evm/abci.go

```diff
@@ -18,6 +18,8 @@ import (
 // TODO: make this a param when we can easily switch between different kinds of keys and different settings
 var keyRole = tss.SecondaryKey
 
+const maxConfirmedEventsPerBlock = 50
```

Comment:
I presume you're going to parameterize this?

Reply:
yes, will do it later tonight